app/controllers/application_controller.rb

```diff
@@ -1851,10 +1856,30 @@ def identify_tl_or_perf_record
     identify_record(params[:id], controller_to_model)
   end
 
-  def assert_privileges(*features)
-    if features.none? { |feature| role_allows?(:feature => feature) }
-      raise MiqException::RbacPrivilegeException, _('The user is not authorized for this task or item.')
-    end
+  def assert_privileges(*features, any: false)
+    msg = "Features checked: #{features.join(', ')}"
```

Comment:
Is it useful to log the features we checked?

Reply:
See the PR description for examples of the features being checked and passing or failing.